server/src/schema/types/types.js

```diff
@@ -0,0 +1,20 @@
+const {
+  GraphQLObjectType,
+  GraphQLID,
+  GraphQLString,
+} = require('graphql');
+
+const UserType = new GraphQLObjectType({
```

Comment:
Keep the types and resolvers next to each other, each entity should export it's own resolver and types. 

They can be in the different files in the same folder but exported and considered as one entity.

Reply:
changed the project structure to better define the schema.